opfunu/utils/operator.py

```diff
@@ -270,9 +305,9 @@ def zakharov_func(x):
     return np.sum(x ** 2) + temp ** 2 + temp ** 4
 
 
-def levy_func(x):
-    x = np.array(x).ravel()
-    w = 1 + (x - 1) / 4
+def levy_func(x, shift=0.0):
```

Comment:
Found a reference for levy function, looks like the original is correct but that the expectation for the cec function was a 0 optimal shifted version. So went with a shift parameter like the hgbat and happy_cat with a default.